components/dashboard/src/teams/NewTeam.tsx

```diff
@@ -52,12 +52,20 @@ export default function () {
         <div className="flex flex-col w-96 mt-24 mx-auto items-center">
             <h1>New Team</h1>
             <p className="text-gray-500 text-center text-base">
-                Teams allow you to <strong>manage multiple projects</strong>, <strong>group workspaces</strong>, and{" "}
-                <strong>collaborate with your team</strong>.
+                <a href="https://www.gitpod.io/docs/configure/teams" className="gp-link">
+                    Teams
+                </a>{" "}
+                allow you to manage related{" "}
+                <a href="https://www.gitpod.io/docs/configure/projects" className="gp-link">
+                    projects
+                </a>{" "}
+                and collaborate with members.
```

Comment:
Optional suggestion: I find just "members" here a bit confusing, and thought that adding the word "other" might help.

```suggestion
                and collaborate with other members.
```

Reply:
DEAL 🤝